tests/integration-tests/tests/dashboard_and_alarms/test_structured_log_events.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+from typing import Iterable
+
+import boto3
+import pytest
+from clusters_factory import Cluster
+from remote_command_executor import RemoteCommandExecutor
+from retrying import retry
+from time_utils import minutes
+from utils import assert_metrics_has_data, retrieve_metric_data
+
+from tests.dashboard_and_alarms.structured_log_event_utils import assert_that_event_exists
+
+
+@retry(stop_max_attempt_number=8, wait_fixed=minutes(2))
+def _test_cluster_health_metric(metric_names: Iterable[str], cluster: Cluster, region: str):
+    """Test metric value is greater than 0 when the compute node error happens."""
+    logging.info(f"Testing that {metric_names} have data.")
+    response = retrieve_metric_data(cluster.name, metric_names, region)
+    assert_metrics_has_data(response)
+
+
+@pytest.mark.usefixtures("instance", "os", "scheduler")
+def test_custom_compute_action_failure(
+    region,
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+    s3_bucket_factory,
+    scheduler_commands_factory,
+):
+    # Create S3 bucket for pre-install scripts
+    bucket_name = s3_bucket_factory()
+
+    bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
+    bad_script = "on_compute_start_error.sh"
```

Comment:
what about rename it to on_compute_configured_error since here we are testing on node configured

Reply:
Renamed.